python/ucxx/ucxx/_lib/tests/test_endpoint.py

```diff
@@ -108,7 +108,6 @@ def test_close_callback(server_close_callback):
         args=(port, server_close_callback),
     )
     client.start()
-    client.join(timeout=10)
-    server.join(timeout=10)
+    join_processes([client + server], timeout=10)
```

Comment:
**Problem**: The `join_processes` function is called with a list containing the sum of the `client` and `server` process objects, which is incorrect. The function expects a list of process objects.

**Suggested Change**: Pass a list containing the `client` and `server` process objects separately.


**Severity (1 - 4)**: 2 - MINOR

**Line**: 111

```suggestion
    join_processes([client, server], timeout=10)
```

-----

<div align="right">
    <small><i>Generated by <b>Llama 3.1 405b</b></i></small>
</div>
<br />


<div align="right">
  <small>Was this helpful?</small>
  <a href="https://codecritic-feedback.nvidia.com?vote=1&comment_guid=4531c64b-bd4d-47c0-ba60-aed5b8c17bf1&is_github=true" target="_blank">
    <button>👍</button>
  </a>
  <a href="https://codecritic-feedback.nvidia.com?vote=-1&comment_guid=4531c64b-bd4d-47c0-ba60-aed5b8c17bf1&is_github=true" target="_blank">
    <button>👎</button>
  </a>
</div>